### PR TITLE
Update docs to include more detail about pathing.

### PR DESCRIPTION
**Pull request by Mineinjava**

### CODE REVIEW

File: pathing/Path.md
Comment (cmontigel) [inline pathing/Path.md]: It's*

**Review by cmontigel — CHANGES_REQUESTED** [review]
LGTM other than your singular mistake, once that is fixed go ahead and merge. You're welcome for being annoying :D

**Comment by Mineinjava** [comment]
I cannot merge until you approve >>>:(

**Review by cmontigel — APPROVED** [review]
Hehe you know you love commit farming

**Pull request merged**
